src/Kaleidoscope-Hardware-Keyboardio-Imago.h

```diff
@@ -0,0 +1,21 @@
+/* -*- mode: c++ -*-
+ * Kaleidoscope-Hardware-Keyboardio-Imago -- Keyboardio Imago hardware support for Kaleidoscope
+ * Copyright (C) 2018  Keyboard.io, Inc
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "kaleidoscope/device/keyboardio/Imago.h"
```

Comment:
are we going to want to deprecate the old toplevel .h files in the future?

Reply:
I'm not sure we should. For in-tree hardware plugins, they aren't needed. For out-of-tree, they are, otherwise Arduino wouldn't be able to find them. It's easier if in- and out-of-tree plugins are structured the same, so I'd keep the toplevel .h files. Both for hardware and other plugins.